Library/Homebrew/dev-cmd/tests.rb

```diff
@@ -107,6 +107,12 @@ def tests
         ENV.delete(env)
       end
 
+      # Codespaces /tmp is mounted 755 which makes Ruby warn constantly.
+      if (ENV["HOMEBREW_CODESPACES"] == "true") && (HOMEBREW_TEMP.to_s == "/tmp")
```

Comment:
I did an `export HOMEBREW_CODESPACES=1` and this still had `/tmp` permissions errors when I ran `brew tests`. 🤔

```
  2) brew extract when extracting a formula retrieves the specified version of formula
     Failure/Error:
       expect { brew "extract", "testball", target[:name], "--version=0.1" }
         .to output(/^#{path}$/).to_stdout
         .and not_to_output.to_stderr
         .and be_a_success

       expected block to not output to stderr, but output "/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/language/python.rb:10: warning: Insecure world writable dir /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor/bundle/ruby/2.6.0/bin in PATH, mode 040777\n"
     # ./test/dev-cmd/extract_spec.rb:49:in `block (3 levels) in <top (required)>'
     # ./test/support/helper/spec/shared_context/integration_test.rb:49:in `block (2 levels) in <top (required)>'
     # ./test/spec_helper.rb:237:in `block (3 levels) in <top (required)>'
     # ./test/spec_helper.rb:236:in `block (2 levels) in <top (required)>'
     # ./test/spec_helper.rb:96:in `block (2 levels) in <top (required)>'
     # ./vendor/bundle/ruby/2.6.0/gems/rspec-retry-0.6.2/lib/rspec/retry.rb:124:in `block in run'
     # ./vendor/bundle/ruby/2.6.0/gems/rspec-retry-0.6.2/lib/rspec/retry.rb:110:in `loop'
     # ./vendor/bundle/ruby/2.6.0/gems/rspec-retry-0.6.2/lib/rspec/retry.rb:110:in `run'
     # ./vendor/bundle/ruby/2.6.0/gems/rspec-retry-0.6.2/lib/rspec_ext/rspec_ext.rb:12:in `run_with_retry'
     # ./vendor/bundle/ruby/2.6.0/gems/rspec-retry-0.6.2/lib/rspec/retry.rb:37:in `block (2 levels) in setup'
     # ./vendor/bundle/ruby/2.6.0/gems/rspec-wait-0.0.9/lib/rspec/wait.rb:46:in `block (2 levels) in <top (required)>'
```

Reply:
We've never really supported world writable installations, and have `setup-homebrew` adjust that accordingly.

https://github.com/Homebrew/actions/blob/077f79969037d09668739e02eadced826d594bad/setup-homebrew/main.sh#L183

Reply:
@issyl0 https://github.com/Homebrew/brew/blob/73d535a206d86c06e6ebc266351a53e28692dace/.devcontainer/on-create-command.sh#L4-L5 fixes this, not sure why it wasn't run for you here.